stl/inc/xfilesystem_abi.h

```diff
@@ -7,6 +7,9 @@
 #ifndef _XFILESYSTEM_ABI_H
 #define _XFILESYSTEM_ABI_H
 #include <yvals_core.h>
+
+#include <xutility>
+
 #if _STL_COMPILER_PREPROCESSOR

```

Comment:
First, `#include <xutility>` should be placed within `#if _STL_COMPILER_PREPROCESSOR` alongside `#include <cstdint>` etc. (If we're being processed by a non-compiler tool, we want to emit as little code as possible.)

Second, `<xfilesystem_abi.h>` is a [core header][], but `<xutility>` isn't, so this inclusion is not allowed by this system:

https://github.com/microsoft/STL/blob/313aa593223e78b458ac50a5a46357601eebfe00/stl/inc/xfilesystem_abi.h#L1
https://github.com/microsoft/STL/blob/313aa593223e78b458ac50a5a46357601eebfe00/stl/inc/xutility#L1

I think the simplest way to deal with this would be to copy `find_if` here, eliminate the checking/unwrapping/rewrapping, and give it an ugly name. (Not `_Find_if_unchecked` since that's already taken.)

[core header]: https://github.com/microsoft/STL/wiki/The-Difference-Between-Core-And-Non-Core-Headers